crates/astria-sequencer/src/service/consensus.rs

```diff
@@ -151,23 +159,39 @@ impl Consensus {
         })
     }
 
-    #[instrument(skip(self))]
+    #[instrument(skip_all, fields(
+        height = %prepare_proposal.height,
+        tx_count = prepare_proposal.txs.len(),
+        time = %prepare_proposal.time
+    ))]
     async fn handle_prepare_proposal(
         &mut self,
         prepare_proposal: request::PrepareProposal,
     ) -> response::PrepareProposal {
         self.app.prepare_proposal(prepare_proposal).await
     }
 
-    #[instrument(skip(self))]
+    #[instrument(skip_all, fields(
+        height = %process_proposal.height,
+        time = %process_proposal.time,
+        tx_count = process_proposal.txs.len(),
+        proposer = %process_proposal.proposer_address,
+        hash = %telemetry::display::hex(&process_proposal.hash),
+        next_validators_hash = %process_proposal.next_validators_hash,
```

Comment:
We should probably decide whether we want to use `tendermint::Hash: Display` or our `telemetry::display::hex` (for both `next_validators_hash` and `hash`).

Note that their `Display` implementation:
1. is upper hex, and
2. always allocates because they rely on the `hex` crate.

I personally try to avoid unnecessary allocations. so `telemetry::display::hex` as much as possible (and consistently lower-cased) is my preference.